opentelemetry-otlp/Cargo.toml

```diff
@@ -16,17 +16,31 @@ license = "Apache-2.0"
 edition = "2018"
 build = "build.rs"
 
+[lib]
+doctest = false
+
 [package.metadata.docs.rs]
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
 async-trait = "0.1"
 futures = "0.3"
-grpcio = "0.6"
+grpcio = { version = "0.6", optional = true }
 opentelemetry = { version = "0.10", default-features = false, features = ["trace"], path = "../opentelemetry" }
-protobuf = "2.18"
+prost = { version = "0.6.1", optional = true }
+protobuf = { version = "2.18", optional = true }
+tonic = { package = "tonic", version = "0.3.1", optional = true }
+tokio = { version = "0.2", features = ["full"], optional = true }
+
+[features]
+default = ["tonic", "tonic-build", "prost"]
```

Comment:
the blocking client should probably be the default for now to match the other exporters (could open a separate issue to change this default if you feel like async exporters should be default)

Reply:
The tonic blocking-client is now enabled by default. User can enable the tonic async-client by using the `async` feature.

Reply:
The tonic blocking-client is now enabled by default. Users can enable the tonic async-client by adding the `async` feature.